Welcome Back,

And it's Day-47 of <#100DaysOfCode>


### Todays Task is to write a Python program to compute the sum of the two reversed numbers and display the sum in reversed form.

In [4]:

def reverse_sum(n1, n2):
    return int(str(int(str(n1)[::-1]) + int(str(n2)[::-1]))[::-1])

In [5]:
reverse_sum(498,198)

5871

In the above task it performing, for example :
    
    n1 = 498 
    n2 = 198
    
    getting reversed :
        n1 =   894
        n2 =   198 
             _____
        sum = 5871

### Write a Python program to check whether a given number is an ugly number.

Ugly numbers are positive numbers whose only prime factors are 2, 3 or 5. The sequence 1, 2, 3, 4, 5, 6, 8, 9, 10, 12, ...
shows the first ugly numbers.
Note: 1 is typically treated as an ugly number.

In [6]:
def is_ugly(num):
        if num == 0:
            return False
        for i in [2, 3, 5]:
            while num % i == 0:
                num /= i
        return num == 1

print(is_ugly(12))
print(is_ugly(13))

True
False
